src/test/java/com/fundacionjala/movies/CustomerTest.java

```diff
@@ -0,0 +1,71 @@
+package com.fundacionjala.movies;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class CustomerTest {
+
+    private Customer customer;
+    private Movie children;
+    private Movie regular;
+    private Movie newRelease;
```

Comment:
Instead of having 3 attributes pointing to the same Movie instance
you can have just one instance because it will be assigned in each test
```java
private Movie movie;
```